app.js

```diff
@@ -20,63 +22,40 @@ console.log(win, lose, tie, resetIt);
 
 // define DOM utitlies
 
-const willYouWin = () => {
-    let rockPaperScissors = ['rock', 'paper', 'scissors'];
-    let rpc = rockPaperScissors[Math.floor(Math.random() * rockPaperScissors.length)];
-    const npcSelection = rpc;
-    console.log(rpc);
+play.addEventListener('click', () => {
+
     const selectedRadioButton = document.querySelector('input:checked');
     const userSelection = selectedRadioButton.value;
-    console.log(userSelection);
-
-    if (userSelection === 'rock' && npcSelection === 'scissors') {
-        win++;
-    }
-    if (userSelection === 'paper' && npcSelection === 'rock') {
-        win++;
-    }
-    if (userSelection === 'scissors' && npcSelection === 'paper') {
-        win++;
-    }
-    if (userSelection === 'rock' && npcSelection === 'paper') {
-        lose++;
-    }
-    if (userSelection === 'paper' && npcSelection === 'scissors') {
-        lose++;
-    }
-    if (userSelection === 'scissors' && npcSelection === 'rock') {
-        lose++;
-    }
-    if (userSelection === 'rock' && npcSelection === 'rock') {
-        tie++;
-    }
-    if (userSelection === 'paper' && npcSelection === 'paper') {
-        tie++;
-    }
-    if (userSelection === 'scissors' && npcSelection === 'scissors') {
-        tie++;
-    }
-
-    // loop back to ui
+    const results = willYouWin(userSelection);
+   
+    if (results === 'win') win++;
+    if (results === 'lose') lose++;
+    if (results === 'tie') tie++;
 
     winSpan.textContent = win;
     lossSpan.textContent = lose;
     tieSpan.textContent = tie;
 
-    console.log(win, lose, tie);
-};
-
-play.addEventListener('click', willYouWin) ;
+});
 
 const hardReset = () => {
     winSpan.textContent = win;
     lossSpan.textContent = lose;
     tieSpan.textContent = tie;
+
+    if (win > 0){
```

Comment:
I think these 'if' statements are going to actually make your reset button function a little strangely.  Try commenting this out.   As well, you'll want to reinitialize all of your state once more, except for your resets :) Give it a try!